scripts/linters/js_ts_linter.py

```diff
@@ -220,6 +228,57 @@ def _compile_ts_file(self, filepath, dir_path):
             dir_path, os.path.basename(filepath).replace('.ts', '.js'))
         return compiled_js_filepath
 
+    def _check_any_type(self):
+        """Checks if the type of any variable is declared as any
+        in typescript files.
+        """
+
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting any type check')
+            python_utils.PRINT('----------------------------------------')
+
+        # First pattern is used to match cases like these : any
+        # Second pattern is used when the character of last line was :
+        # So, we know that this line begins with a type of variable
```

Comment:
```suggestion
        # First pattern is used to match cases like these: ': any'.
        # Second pattern is used when the last character of 
        # the previous line was ':' so that we know that this line
        # begins with a type.
```

Reply:
Done